packages/spright-components/src/chat/conversation/styles.ts

```diff
@@ -18,8 +22,19 @@ export const styles = css`
         flex-direction: column;
         justify-content: flex-start;
         row-gap: 32px;
-        padding: ${mediumPadding};
-        background: ${applicationBackgroundColor};
+        padding: ${mediumPadding} ${standardPadding} ${mediumPadding}
+            ${standardPadding};
+        border: ${borderWidth} solid ${applicationBackgroundColor};
+        background: ${sectionBackgroundImage};
         overflow-y: auto;
     }
+
+    .input {
+        padding: ${mediumPadding} ${standardPadding} ${standardPadding}
```

Comment:
mentioned in sb comment but the top padding here ends up clipping messages in a weird way / in mid air
![Uploading image.png…]()

Reply:
I chatted with Brandon and we decided to shrink that padding down to 1px for now.

![image](https://github.com/user-attachments/assets/f37d29df-7a58-4d1d-9be7-9d2414cb2680)